libs/island-ui/core/src/lib/Link/Link.tsx

```diff
@@ -78,6 +80,7 @@ export const Link: React.FC<LinkProps> = ({
           <a
             className={classNames}
             {...linkProps}
+            {...(newTab && { target: '_blank' })}
```

Comment:
I would use it along `rel="noopener noreferrer"`

Reply:
Is it useful for internal links that open in a new tab? This link is within a `  if (isInternal) ...` block. External links already get `rel="noopener noreferrer"`.

Reply:
Good point, didn't see it was for internal links 👍